next/components/pages/InbaArticlePageContent.tsx

```diff
@@ -53,13 +57,20 @@ const InbaArticlePageContent = ({ inbaArticle }: InbaArticlePageContentProps) =>
           </NarrowText>
         </SectionContainer>
       ) : null}
-
       <SectionContainer className="pt-10 md:pt-18">
         <NarrowText align="left" width="wide">
           <Markdown content={inbaArticle.attributes?.content} />
+          {/* TODO set padding-top according to styleguide */}
```

Comment:
Yes, please add some padding :D and use the same font size as is in Markdown please.